src/lib/document-matcher/type-manager/types/geo.ts

```diff
@@ -78,6 +80,13 @@ interface geoDistance {
     unit: Units;
 }
 
+type geoPointArr = [number, number];
+type geoPointStr = string;
+type geoObj1 = {lat: string | number, lon: string | number};
```

Comment:
It is weird to this `geoObj1`, it is better to use more descriptive naming.

```suggestion
type geoObjShort = {lat: string | number, lon: string | number};
```